cmd/install/assets/hypershift_operator.go

```diff
@@ -802,7 +802,7 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 			},
 			{
 				APIGroups: []string{"apps"},
-				Resources: []string{"deployments", "statefulsets"},
+				Resources: []string{"deployments", "replicasets", "statefulsets"},
```

Comment:
why is replicasets needed?

Reply:
It's in the commit message
> CSI snapshot controller operator requires "get" on ReplicaSets. Granting
everything to be in sync with Deployments.

The operator gets ReplicaSet to get its owner (Deployment) to be able to create all its Events with the Deployment as the involvedObject.

Reply:
note: This will require updating the role on the root HO in the CI cluster. cc @csrwng @sjenning